configure.ac

```diff
@@ -2334,6 +2334,9 @@ yes)
 no)
   IRDP=false
   ;;
+*)
```

Comment:
What value is this covering for? Would hope that AC_ARG_ENABLE() macro would produce deterministic results (i.e., only `yes` or `no`).

Reply:
It is neither `yes` not `no` if there's no `--enable/disable-irdp` argument provided. So it doesn't hit any of the two existing cases. It was ok before, because the default was to enable the feature, but now we have to disable it by default.